apps/cookbook/src/app/examples/item-example/examples/disclosure-animation.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-item-example-disclosure-animation',
+  template: `<kirby-item selectable="true" [disclosure]="'arrow-down'" [rotateIcon]="rotateIcon" (click)="rotateIcon = !rotateIcon">
```

Comment:
Consider simplifying as it's just a string:
```suggestion
  template: `<kirby-item selectable="true" disclosure="arrow-down" [rotateIcon]="rotateIcon" (click)="rotateIcon = !rotateIcon">
```